lib/extensions/association_inverse_suppression/active_record/base.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 module Extensions::AssociationInverseSuppression::ActiveRecord::Base
   module ClassMethods
+    # Bring forward the userstamp association definitions
+    def inherited(klass)
```

Comment:
I noticed that there's already an `inherited` callback in `activerecord-userstamp-3.0.4/lib/active_record/userstamp/stampable.rb`, why need to declare here again? because of the execution order?

Reply:
The `inherited` callback was removed in `3.0.5`. Adding it back here.

Reply:
I see. but why it's still needed? I think we should figure out the reason and probably should do a fix to the upstream.